src/Event/Productions/SimilaritiesClient.php

```diff
@@ -62,4 +63,19 @@ public function excludePermanently(SimilarEventPair $pair)
             ['json' => $data]
         );
     }
+
+    public function nextSuggestion(\DateTime $dateFrom, int $size = 1, int $offset = 0): Suggestion
```

Comment:
Nice, a good idea to already include a `size` parameter in case we do end up fetching multiples at once down the road depending on how the performance improvements go.